_pages/welcome-to-TTS/classes/working-groups-and-guilds-101.md

```diff
@@ -177,10 +177,10 @@ The open Slack channel for guild leaders & other practice leaders who wish to jo
         </td>
       </tr>
       <tr>
-        <td class="col-grouplet"><strong><a id="user-research">User Research</a></strong></td>
+        <td class="col-grouplet"><strong><a id="design-research">Design Research</a></strong></td>
```

Comment:
I'm betting something is referring to this ID, probably as an anchor link, and we need to update that, too, wherever it is. 

Also, I'd like to suggest rather than using a malformed link we pull the ID up to the strong element or the td. We'll get the same result without the broken HTML.